### PR TITLE
Add loop toggle checkbox

### LINKED ISSUE #144: Implement looped playback toggling

### PR DESCRIPTION
**Pull request by ConnorJacksonBSU**
closes #144

### CODE REVIEW

**Review by PTLux1517 — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ export default function App() {
 	const [playing, setPlaying] = useState(false);
 	const [frameStart, setStart] = useState(0);
 	const [frameEnd, setEnd] = useState(0);
+	const [checked, setChecked] = useState(true);
```
Comment (vpzomtrrfrt) [inline src/App.tsx:23]: should have a more specific name

File: src/App.tsx [hunk 2]
```diff
@@ -137,14 +141,13 @@ export default function App() {
 	const togglePlaying = useCallback(() => {
 		setPlaying(current => {
 			if(current) return false;
-
-			if(frameRef.current >= markerFileData.frames.length - 1) setFrame(frameStart); // restart if at end
+			if(checked == true && frameRef.current >= markerFileData.frames.length - 1) setFrame(frameStart); // restart if at end
```
Comment (vpzomtrrfrt) [inline src/App.tsx:144]: not sure the checkbox should affect this, since if we don't reset then it won't play anything

Comment (ConnorJacksonBSU) [reply src/App.tsx:144]: right I forgot to remove it, was testing some things

Comment (ConnorJacksonBSU) [reply src/App.tsx]: taken out

File: src/App.tsx (hunk 1 above)
Comment (ConnorJacksonBSU) [reply src/App.tsx]: resolved

File: src/App.tsx [hunk 3]
```diff
@@ -116,9 +120,12 @@ export default function App() {
 					interFrameTimeRef.current -= timeStep;
 					setFrame(current => {
 						if(current + 1 < markerFileData.frames.length && current + 1 < frameEnd) {return current + 1;
-						} else return frameStart;
-						//setPlaying(false);
-						//return current;
+						} else if(loopCheck == false) { //if loop checkbox unchecked then don't loop
+								setPlaying(loopCheck); //changed play to pause once reaches end if loop unchecked, for frame manipulation
```
Comment (vpzomtrrfrt) [inline src/App.tsx:124]: seems weird to pass this variable since you already know it's false

File: src/App.tsx [hunk 4]
```diff
@@ -203,6 +209,13 @@ export default function App() {
 				</button>
 				<input id={"timeline-track"} type={"range"} min={"0"} max={markerFileData.frames.length - 1}
 					onChange={(e) => setFrame(parseInt(e.target.value))} value={frame} />
+				<label><input type="checkbox" checked={loopCheck} onChange={(e) => {
+					setLoopCheck(e.target.checked);
+					/*if (!checked) {
```
Comment (vpzomtrrfrt) [inline src/App.tsx:214]: do we want this or not? commented out code isn't very nice

**Comment by PTLux1517** [comment]
This feature has been migrated into the refactor branch (see PR #177)

**Pull request closed without merging**